### PR TITLE
[Doc update][Code section]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Please review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c78e4afe8a2c0493046fd819155cf831267617e8/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 97 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6a21f817-cbd0-440f-95a0-c3f1d45ce2bc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/452e6343ebd709afe7c4174f113ab1aa7f25ac3a/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 97 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eed6a015-cd74-4c9d-9422-0fba070edf4b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md [hunk 1]
```diff
@@ -474,22 +465,13 @@ To deserialize the JSON payload, a model object is introduced to get the metadat
 > * Do not use conversation ID as meeting ID.     
 > * Do not use meeting ID from meeting events payload `turncontext.activity.value`. 
       
-The following code shows how to capture the metadata of a meeting that is `MeetingType`, `Title`, `Id`, `JoinUrl`, `StartTime`, and `EndTime` from a meeting start and end event:
+The following code shows how to capture the metadata of a meeting that is `MeetingType`, `Title`, `Id`, `JoinUrl`, `StartTime`, and `EndTime` from a meeting start event:
```
Comment (v-abirade) [inline msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md:468]: As updated by Yunny:
The following code shows how to capture the metadata of a meeting that is `MeetingType`, `Title`, `Id`, `JoinUrl`, `StartTime`, and `EndTime` from a meeting start/end event:
As per MSTP:
```suggestion: The following code shows how to capture the metadata of a meeting that is `MeetingType`, `Title`, `Id`, `JoinUrl`, `StartTime`, and `EndTime` from a meeting start and end event.
```

**Review by v-abirade — CHANGES_REQUESTED** [review]
Please make changes as per the link: https://github.com/MicrosoftDocs/msteams-docs/pull/3957/files
All the changes are not reflecting. Also refer to the comment please.
Please re-work on this and share in the evening call.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e0be98aeecbac48ce4ab809f1c16ac1e71283a71/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 97 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7bfe343d-90d4-4e1b-b713-110f277a6cec) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/deeeda7f175571c1c18b9cb6443283c1fb4ee0fd/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 97 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1afa4403-2edd-44ba-8e82-523b666c3716) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by YunnyChung — APPROVED** [review]
Thanks for making changes, Rajesh!

**Review by v-abirade — APPROVED** [review]
Approved

**Pull request merged**